src/design-documentation/project-structure.md

```diff
@@ -52,16 +52,41 @@
   - **unit** — Unit tests using rustc’s built-in test harness.
   - **wpt** — W3C web-platform-tests and csswg-tests along with tools to run them and expected failures.
 
-# Major dependencies
+# Repositories the Servo project maintains
 
-* <https://github.com/servo/mozjs>: bindings to SpiderMonkey
-* <https://github.com/hyperium/hyper>: an HTTP implementation
-* <https://github.com/servo/html5ever>: an HTML5 parser
-* <https://github.com/servo/ipc-channel>: an IPC implementation
-* <https://github.com/image-rs/image>: image decoders
-* <https://github.com/rust-windowing/winit>: cross-platform windowing and input
-* <https://github.com/linebender/vello>: a pure Rust 2D graphics library
-* <https://github.com/servo/rust-cssparser>: a CSS parser
-* <https://github.com/housleyjk/ws-rs>: a WebSocket protocol implementation
-* <https://github.com/servo/rust-url>: an implementation of the URL specification
-* <https://github.com/servo/webrender>: a GPU renderer
+The Servo project maintains a number of repositories that are either independent from Servo or forked from some upstream project with ongoing updates.
+
+## Repositories that are used throughout the rust ecosystem
+
+- [ipc-channel](https://github.com/servo/ipc-channel): Crate for the rust ecosystem of making IPC work.
+- [string-cache](https://github.com/servo/string-cache): String interning library.
```

Comment:
Nit: remove the period on this line and the rest, unless the fragment is being separated from another complete sentence. Periods don't usually follow sentence fragments like this.

```suggestion
- [string-cache](https://github.com/servo/string-cache): String interning library
```